components/chainhook-sdk/src/chainhooks/stacks/mod.rs

```diff
@@ -829,6 +829,7 @@ pub fn handle_stacks_hook_action<'a>(
                 client
                     .request(method, &host)
                     .header("Content-Type", "application/json")
+                    .header("Authorization", http.authorization_header.clone())
```

Comment:
If we aren't already logging these backed by a DEBUG flag or equivalent, and if it is not an expensive operation, could we consider logging what's being packaged up as part of the response or server-side re-direct?

Reply:
So I realize if the redirect fails for any reason, that will be caught, and the details of what was attempted as the URL (including the header details) will be included in that error message, which is sufficient after all.

We don't have to explicitly log any further details, and with that, I will resolve this comment.